pkg/api/mlflow/dao/models/relation.go

```diff
@@ -0,0 +1,19 @@
+package models
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+// RoleNamespace represents model to work with `role_relations` table.
+// Model holds relations between Role and Namespace models.
+type RoleNamespace struct {
```

Comment:
we should probably embed `Base` here to avoid duplicating `ID`, `CreatedAt`, and `UpdatedAt`. Probably `IsArchived` is not desirable in this case and be moved into a `ArchivableBase` struct.

Reply:
i know we have the plan to redo the schema at some point -- at that point we can probably arrive at a uniform base model.